Modules/Sources/WordPressCore/Plugins/PluginService.swift

```diff
@@ -38,6 +39,18 @@ public actor PluginService: PluginServiceProtocol {
         return nil
     }
 
+    public func togglePluginActivation(slug: PluginSlug) async throws {
+        let plugin = try await client.api.plugins.retrieveWithViewContext(pluginSlug: slug)
```

Comment:
Should it use state from the View? It would be a bit confusing if the view shows "active", you tap "deactivate", and it ends up activating it. Is that something that can happen now?

Reply:
I plan to change the `InstalledPlugin` type, which would be a fairly large changeset. I'll follow up in a separate PR.